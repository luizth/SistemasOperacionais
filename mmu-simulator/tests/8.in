6
2 100
3 13
5 9
2 92
2 71
2 82
0 32
4 105
0 21
2 87
0 110
3 12
5 85
4 104
0 28
1 121
4 72
0 65
1 101
4 29
0 86
0 10
1 23
0 110
3 3
0 106
2 73
4 97
1 47
0 32
0 102
0 62
4 72
2 115
3 77
1 67
1 54
3 12
1 43
4 36
1 47
2 114
4 63
1 43
1 27
4 14
0 62
4 94
0 86
4 97
3 64
0 21
3 66
5 44
4 36
0 32
0 69
4 4
5 51
0 122
1 47
2 68
3 12
3 53
4 29
3 91
3 89
1 34
5 9
3 126
0 84
3 31
1 67
3 31
1 43
4 59
2 26
1 20
4 56
2 17
3 52
1 25
2 113
1 54
5 55
5 85
1 27
3 81
2 42
2 99
2 42
0 62
2 87
4 7
0 84
1 25
1 20
