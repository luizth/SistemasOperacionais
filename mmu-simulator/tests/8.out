Process 2 accessed page 100 - frame 0
Process 3 accessed page 13 - frame 1
Process 5 accessed page 9 - frame 2
Process 2 accessed page 92 - frame 3
Process 2 accessed page 71 - frame 4
Process 2 accessed page 82 - frame 5
Process 0 accessed page 32 - frame 6
Process 4 accessed page 105 - frame 7
Memory is full! Frame 0 was replaced (page 100 - pid 2)
Process 0 accessed page 21 - frame 0
Memory is full! Frame 1 was replaced (page 13 - pid 3)
Process 2 accessed page 87 - frame 1
Memory is full! Frame 2 was replaced (page 9 - pid 5)
Process 0 accessed page 110 - frame 2
Memory is full! Frame 3 was replaced (page 92 - pid 2)
Process 3 accessed page 12 - frame 3
Memory is full! Frame 4 was replaced (page 71 - pid 2)
Process 5 accessed page 85 - frame 4
Memory is full! Frame 5 was replaced (page 82 - pid 2)
Process 4 accessed page 104 - frame 5
Memory is full! Frame 6 was replaced (page 32 - pid 0)
Process 0 accessed page 28 - frame 6
Memory is full! Frame 7 was replaced (page 105 - pid 4)
Process 1 accessed page 121 - frame 7
Memory is full! Frame 0 was replaced (page 21 - pid 0)
Process 4 accessed page 72 - frame 0
Memory is full! Frame 1 was replaced (page 87 - pid 2)
Process 0 accessed page 65 - frame 1
Memory is full! Frame 2 was replaced (page 110 - pid 0)
Process 1 accessed page 101 - frame 2
Memory is full! Frame 3 was replaced (page 12 - pid 3)
Process 4 accessed page 29 - frame 3
Memory is full! Frame 4 was replaced (page 85 - pid 5)
Process 0 accessed page 86 - frame 4
Memory is full! Frame 5 was replaced (page 104 - pid 4)
Process 0 accessed page 10 - frame 5
Memory is full! Frame 6 was replaced (page 28 - pid 0)
Process 1 accessed page 23 - frame 6
Memory is full! Frame 7 was replaced (page 121 - pid 1)
Process 0 accessed page 110 - frame 7
Memory is full! Frame 0 was replaced (page 72 - pid 4)
Process 3 accessed page 3 - frame 0
Memory is full! Frame 1 was replaced (page 65 - pid 0)
Process 0 accessed page 106 - frame 1
Memory is full! Frame 2 was replaced (page 101 - pid 1)
Process 2 accessed page 73 - frame 2
Memory is full! Frame 3 was replaced (page 29 - pid 4)
Process 4 accessed page 97 - frame 3
Memory is full! Frame 4 was replaced (page 86 - pid 0)
Process 1 accessed page 47 - frame 4
Memory is full! Frame 5 was replaced (page 10 - pid 0)
Process 0 accessed page 32 - frame 5
Memory is full! Frame 6 was replaced (page 23 - pid 1)
Process 0 accessed page 102 - frame 6
Memory is full! Frame 7 was replaced (page 110 - pid 0)
Process 0 accessed page 62 - frame 7
Memory is full! Frame 0 was replaced (page 3 - pid 3)
Process 4 accessed page 72 - frame 0
Memory is full! Frame 1 was replaced (page 106 - pid 0)
Process 2 accessed page 115 - frame 1
Memory is full! Frame 2 was replaced (page 73 - pid 2)
Process 3 accessed page 77 - frame 2
Memory is full! Frame 3 was replaced (page 97 - pid 4)
Process 1 accessed page 67 - frame 3
Memory is full! Frame 4 was replaced (page 47 - pid 1)
Process 1 accessed page 54 - frame 4
Memory is full! Frame 5 was replaced (page 32 - pid 0)
Process 3 accessed page 12 - frame 5
Memory is full! Frame 6 was replaced (page 102 - pid 0)
Process 1 accessed page 43 - frame 6
Memory is full! Frame 7 was replaced (page 62 - pid 0)
Process 4 accessed page 36 - frame 7
Memory is full! Frame 0 was replaced (page 72 - pid 4)
Process 1 accessed page 47 - frame 0
Memory is full! Frame 1 was replaced (page 115 - pid 2)
Process 2 accessed page 114 - frame 1
Memory is full! Frame 2 was replaced (page 77 - pid 3)
Process 4 accessed page 63 - frame 2
Page 43 in memory (frame 6)!
Process 1 accessed page 43 - frame 6
Memory is full! Frame 3 was replaced (page 67 - pid 1)
Process 1 accessed page 27 - frame 3
Memory is full! Frame 4 was replaced (page 54 - pid 1)
Process 4 accessed page 14 - frame 4
Memory is full! Frame 5 was replaced (page 12 - pid 3)
Process 0 accessed page 62 - frame 5
Memory is full! Frame 6 was replaced (page 43 - pid 1)
Process 4 accessed page 94 - frame 6
Memory is full! Frame 7 was replaced (page 36 - pid 4)
Process 0 accessed page 86 - frame 7
Memory is full! Frame 0 was replaced (page 47 - pid 1)
Process 4 accessed page 97 - frame 0
Memory is full! Frame 1 was replaced (page 114 - pid 2)
Process 3 accessed page 64 - frame 1
Memory is full! Frame 2 was replaced (page 63 - pid 4)
Process 0 accessed page 21 - frame 2
Memory is full! Frame 3 was replaced (page 27 - pid 1)
Process 3 accessed page 66 - frame 3
Memory is full! Frame 4 was replaced (page 14 - pid 4)
Process 5 accessed page 44 - frame 4
Memory is full! Frame 5 was replaced (page 62 - pid 0)
Process 4 accessed page 36 - frame 5
Memory is full! Frame 6 was replaced (page 94 - pid 4)
Process 0 accessed page 32 - frame 6
Memory is full! Frame 7 was replaced (page 86 - pid 0)
Process 0 accessed page 69 - frame 7
Memory is full! Frame 0 was replaced (page 97 - pid 4)
Process 4 accessed page 4 - frame 0
Memory is full! Frame 1 was replaced (page 64 - pid 3)
Process 5 accessed page 51 - frame 1
Memory is full! Frame 2 was replaced (page 21 - pid 0)
Process 0 accessed page 122 - frame 2
Memory is full! Frame 3 was replaced (page 66 - pid 3)
Process 1 accessed page 47 - frame 3
Memory is full! Frame 4 was replaced (page 44 - pid 5)
Process 2 accessed page 68 - frame 4
Memory is full! Frame 5 was replaced (page 36 - pid 4)
Process 3 accessed page 12 - frame 5
Memory is full! Frame 6 was replaced (page 32 - pid 0)
Process 3 accessed page 53 - frame 6
Memory is full! Frame 7 was replaced (page 69 - pid 0)
Process 4 accessed page 29 - frame 7
Memory is full! Frame 0 was replaced (page 4 - pid 4)
Process 3 accessed page 91 - frame 0
Memory is full! Frame 1 was replaced (page 51 - pid 5)
Process 3 accessed page 89 - frame 1
Memory is full! Frame 2 was replaced (page 122 - pid 0)
Process 1 accessed page 34 - frame 2
Memory is full! Frame 3 was replaced (page 47 - pid 1)
Process 5 accessed page 9 - frame 3
Memory is full! Frame 4 was replaced (page 68 - pid 2)
Process 3 accessed page 126 - frame 4
Memory is full! Frame 5 was replaced (page 12 - pid 3)
Process 0 accessed page 84 - frame 5
Memory is full! Frame 6 was replaced (page 53 - pid 3)
Process 3 accessed page 31 - frame 6
Memory is full! Frame 7 was replaced (page 29 - pid 4)
Process 1 accessed page 67 - frame 7
Page 31 in memory (frame 6)!
Process 3 accessed page 31 - frame 6
Memory is full! Frame 0 was replaced (page 91 - pid 3)
Process 1 accessed page 43 - frame 0
Memory is full! Frame 1 was replaced (page 89 - pid 3)
Process 4 accessed page 59 - frame 1
Memory is full! Frame 2 was replaced (page 34 - pid 1)
Process 2 accessed page 26 - frame 2
Memory is full! Frame 3 was replaced (page 9 - pid 5)
Process 1 accessed page 20 - frame 3
Memory is full! Frame 4 was replaced (page 126 - pid 3)
Process 4 accessed page 56 - frame 4
Memory is full! Frame 5 was replaced (page 84 - pid 0)
Process 2 accessed page 17 - frame 5
Memory is full! Frame 6 was replaced (page 31 - pid 3)
Process 3 accessed page 52 - frame 6
Memory is full! Frame 7 was replaced (page 67 - pid 1)
Process 1 accessed page 25 - frame 7
Memory is full! Frame 0 was replaced (page 43 - pid 1)
Process 2 accessed page 113 - frame 0
Memory is full! Frame 1 was replaced (page 59 - pid 4)
Process 1 accessed page 54 - frame 1
Memory is full! Frame 2 was replaced (page 26 - pid 2)
Process 5 accessed page 55 - frame 2
Memory is full! Frame 3 was replaced (page 20 - pid 1)
Process 5 accessed page 85 - frame 3
Memory is full! Frame 4 was replaced (page 56 - pid 4)
Process 1 accessed page 27 - frame 4
Memory is full! Frame 5 was replaced (page 17 - pid 2)
Process 3 accessed page 81 - frame 5
Memory is full! Frame 6 was replaced (page 52 - pid 3)
Process 2 accessed page 42 - frame 6
Memory is full! Frame 7 was replaced (page 25 - pid 1)
Process 2 accessed page 99 - frame 7
Page 42 in memory (frame 6)!
Process 2 accessed page 42 - frame 6
Memory is full! Frame 0 was replaced (page 113 - pid 2)
Process 0 accessed page 62 - frame 0
Memory is full! Frame 1 was replaced (page 54 - pid 1)
Process 2 accessed page 87 - frame 1
Memory is full! Frame 2 was replaced (page 55 - pid 5)
Process 4 accessed page 7 - frame 2
Memory is full! Frame 3 was replaced (page 85 - pid 5)
Process 0 accessed page 84 - frame 3
Memory is full! Frame 4 was replaced (page 27 - pid 1)
Process 1 accessed page 25 - frame 4
Memory is full! Frame 5 was replaced (page 81 - pid 3)
Process 1 accessed page 20 - frame 5
